service/src/main/java/com/derpgroup/derpwizard/manager/AbstractManager.java

```diff
@@ -64,6 +82,31 @@ public void handleRequest(@NonNull VoiceInput voiceInput, @NonNull SsmlDocumentB
     }
   }
 
+  private void appendToConversationHistory(@NonNull VoiceInput voiceInput) {
```

Comment:
This entire function can be moved into the ConversationHistoryUtils class as we've just talked about.